modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
so how about naming it different way, so that we detect/avoid errors if we pass some object from jitsi by mistake ?